app/javascript/src/Policies/actions/index.jsx

```diff
@@ -96,7 +93,20 @@ function closePolicyConfig (): ThunkAction {
   }
 }
 
-export {
+export interface IPoliciesActions {
```

Comment:
You did not change the policies right?

Reply:
You can ignore that. I was in the middle of a rebasing. This file is no longer listed in the changes.